config/logstash/logstash.conf

```diff
@@ -0,0 +1,51 @@
+input {
+  jdbc {
+    jdbc_driver_library => "/usr/share/logstash/postgresql-42.6.0.jar"
+    jdbc_driver_class => "org.postgresql.Driver"
+    jdbc_connection_string => "jdbc:postgresql://db/postgres"
+    jdbc_user => "postgres"
+    jdbc_password => "mysecretpassword"
+    schedule => "* * * * *"

```

Comment:
I'd probably just add a `# Every 1 minute` note here so folks who don't know cron as well know what this means!